inclusive-design/idrc
# GH-212: add thumbnail to projects and tools (Resolves #212)

## Description

Add the options for supplying thumbnails to the projects and tools page, similar to the ideas page.

## Steps to test

1. "projects and tools" page should load with thumbnails for some of the projects
2. going to the admin page, should show an option to add thumbnails to the projects and tools pages.

## Additional information

There is currently a build error that needs to be investigated that is preventing the site from being built.

`unknown block tag: getUrl` Related to generating the thumbnails. It's unclear why this doesn't work as the shortcode works on the ideas page.

## Related issues

Resolves #212
